company-profiles/beyondcompany.md

```diff
@@ -0,0 +1,38 @@
+# BeyondPricing
+
+## Company blurb
+
+Beyond Company (aka Beyond Co.) is a product company (startup like). We use agile development and cutting edge technology when creating solutions for our customers. Counting on developers, designers and even jurists, our multidisciplinary team offers services such as Artificial Intelligence, G Suite Migration and Customization, Cloud Architecture and Development, evaluating large volumes of data and obtaining great performances in our clients' businesses. More info in our [website](https://beyondcompany.com.br).
+
+## Company size
+
+20-50
+
+## Remote status
+
+All the developers are remote. We also have an offices spread across several places in Brazil..
+
+## Region
+
+- Brazil
+
+## Company technologies
+
+Our main stack is:
+- VueJS
+- Javascript/Typescript
+- Sequelize + Express/NestJS
+
+No specific language experience is required. If you have experience in similar fields we want to hear from you.
+
+## Office locations
+
+- Recife - Brazil
+- Rio de Janeiro - Brazil
+- São Paulo - Brazil
+- Belo Horizonte - Brazil
+- Brasília - Brazil
+
+## How to apply
+
+Send email to curriculo+githubremotejobs@beyondcompany.com.br
```

Comment:
Is there a website for careers, if not is `curriculo+githubremotejobs@beyondcompany.com.br` correct, specifically the `.com.br`?

Reply:
No. Currently `beyondcompany.com.br` is only a landing page. Currently we accept applying only by this mailbox.